src/daemon/daemonlocalserverconnection.cpp

```diff
@@ -160,3 +171,60 @@ void DaemonLocalServerConnection::write(const QJsonObject& obj) {
   m_socket->write(QJsonDocument(obj).toJson(QJsonDocument::Compact));
   m_socket->write("\n");
 }
+
+bool DaemonLocalServerConnection::isCallerAuthorized(const QString& command) {
+#if defined(MZ_MACOS)
```

Comment:
Nevermind they jsut stub it. https://github.com/postgres/postgres/blob/2406c4e34ccca697bd5a221f8375f335b5841dea/doc/src/sgml/client-auth.sgml#L1726-L1732.

Reply:
Calling on @vinocher . Should we file a follow up to do this on Windows? Or do you think this is not even possible?

Reply:
can we use [this code](https://github.com/postgres/postgres/blob/master/src/port/getpeereid.c) for Windows/platforms without `peereid()`? I think it's unix specific but just in case I'm missing something

Reply:
That code would fallback to the "no implementation available" branch on Windows.